LICENSE

```diff
@@ -217,3 +200,177 @@ solely pursuant to the terms of the relevant commercial agreement.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+================================================================================
```

Comment:
i suggest a consistent delimiter. either 80 = or sth. like ```---``` and two newlines above?